.github/workflows/test.yml

```
@@ -50,15 +89,16 @@ jobs:
           mkdir qemu-data
```

```suggestion
          mkdir -p qemu-data
```

Currently this line would fail if folder already exists.

---

0fb47d0128d6b1601e1d9bc018b68cf76406519f